src/synthesizer/components/stellar.py

```diff
@@ -197,7 +197,7 @@ def get_spectra_incident(
         sed = Sed(grid.lam, lnu)
 
         # Store the Sed object
-        self.spectra[label + "incident"] = sed
+        self.spectra[f"{label}_incident"] = sed
```

Comment:
I'm not sure these fixes are right? Doesn't `label` include a trailing underscore by design? Explicitly because you can also have a incident on it's own? Or is that not true for CFOO?

Reply:
Oh wait, I just the change below. Is this specifically because you always have young and old spectra in CF00?

Reply:
Actually, you're absolutely right. Commit below fixes this